app.json

```diff
@@ -0,0 +1,10 @@
+{
+  "addons": [
+    "logentries:entry_plus"
```

Comment:
Good idea, I didn't know Heroku would choose the free tier if we aren't explicit about it